seahorn/include/utils.h

```diff
@@ -10,6 +10,16 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#ifndef __KLEE__ // define macros for KLEE
+#define __KLEE__ 0
```

Comment:
remove. if not defined then do not define as anything else

Reply:
Will remove in upcoming commits.